### PR TITLE
Our values - foundation for code of conduct

### LINKED ISSUE #14: Structure our code of conduct
**Type of Issue**  
- [x] Addition / Enhancement / Improvement

**Type of Work**  
- [x] Documentation

**Description**  
Our code of conduct follows the standard definition:
a set of rules outlining the social norms, rules, and responsibilities or proper practices of an individual party or an organisation.

As a member of ScaLabs,  
I want to know what my duties are in practicing the organisation values, so that I invest my resources effectively.  

As a partner/client of ScaLabs,  
I want to know what qualities I can expect from ScaLabs' team & their services

**Actionable Tasks**  
- [ ] 

**Acceptance criteria**  

To look:
https://abc.xyz/investor/other/google-code-of-conduct/

### PR DESCRIPTION
**Pull request by armantorkzaban**

### CODE REVIEW

**Comment by armantorkzaban** [comment]
@lukas-h, 
we need to write our values first and then code of conduct. Check & share your comments: https://github.com/scalabs/documentation/pull/15/commits/ab36451f4ad4de9e4141b613914e9db2981a65ed

File: VALUES-CODE-OF-CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- Less but better
+- Form follows the content
+- Committment to highest standards defines us
+
+Topics:
+- Serve
+- Respect
+- Conflict resolution
+- Dealing the competitor 
+- ...
```
Comment (lukas-h) [inline VALUES-CODE-OF-CONDUCT.md]: How to communicate and behave with colleagues, clients, and external partners is really important and should be the one of the main topics here

Comment (armantorkzaban) [reply VALUES-CODE-OF-CONDUCT.md]: How about we extend the Respect with those items as sub-categories like this?:
```
Respect:
our colleagues
--1.
--2.
our clients
-communication
--1.
--2.

```

File: VALUES-CODE-OF-CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+## Values
+- ScaLabs is both labs & people
+- Functioning societies produce wonder
+- Human rights are universal
+- Good is green
+- Less but better
+- Form follows the content
+- Committment to highest standards defines us
+
+Code of conduct:
```
Comment (lukas-h) [inline VALUES-CODE-OF-CONDUCT.md]: Don't judge colleagues when they ask "stupid" questions. We want an open discourse-positive culture

File: VALUES-CODE-OF-CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+Our code of conduct are the measures which ensure that we stick to our values
+at every stage of our work and all over at our organisation.
+
+
+## Values
+- ScaLabs is both labs & people
+- Functioning societies produce wonder
+- Human rights are universal
+- Good is green
```
Comment (lukas-h) [inline VALUES-CODE-OF-CONDUCT.md]: Sustainability should be highlighted

File: VALUES-CODE-OF-CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+- Less but better
+- Form follows the content
+- Committment to highest standards defines us
+
```
Comment (lukas-h) [inline VALUES-CODE-OF-CONDUCT.md]: Our quality aspirations:
- tested
- good security
- as good as possible

File: VALUES-CODE-OF-CONDUCT.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+Our code of conduct are the measures which ensure that we stick to our values
+at every stage of our work and all over at our organisation.
+
+
```
Comment (lukas-h) [inline VALUES-CODE-OF-CONDUCT.md]: Open and proactive communication:
Regarding
- time
- feelings
- solutions to problems
- ideas
- ...

File: VALUES-CODE-OF-CONDUCT.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+## Values
+- ScaLabs is both labs & people
+- Functioning societies produce wonder
```
Comment (lukas-h) [inline VALUES-CODE-OF-CONDUCT.md]: - We as an organization are super-national
- Catching possibilities beyond administrative borders

File: VALUES-CODE-OF-CONDUCT.md (hunk 4 above)
Comment (lukas-h) [inline VALUES-CODE-OF-CONDUCT.md]: good will: We find the best solutions for customers

File: VALUES-CODE-OF-CONDUCT.md (hunk 1 above)
Comment (lukas-h) [reply VALUES-CODE-OF-CONDUCT.md]: Could voice and tone guidelines be part of this?

**Comment by lukas-h** [comment]
I think, we should bring this into our next "Governance & Documentation" meeting on Tuesday for further discussion

**Pull request closed without merging**